_raml/apps/products/products.raml

```diff
@@ -33,6 +33,10 @@ get:
         Other query parameters, such as sort, categoryId, page etc, are not supported.
       required: false
       repeat: true
+    includeInvisible:
+      description: Displays invisible products. Only available with `products_write` authorisation.
```

Comment:
Better
*Include invisible products* or
*Display invsibible products as well*
to make clear that the non-invisible products are shown in all cases